Python/libraries/recognizers-date-time/recognizers_date_time/date_time/constants.py

```diff
@@ -71,16 +177,20 @@ class TimeTypeConstants:
     END: str = 'end'
 
     # Mod Value
-    # "before" -> To mean "preceding in time". I.e. Does not include the extracted datetime entity in the resolution's ending point. Equivalent to "<"
+    # "before" -> To mean "preceding in time". I.e.
+    # Does not include the extracted datetime entity in the resolution's ending point. Equivalent to "<"
     BEFORE_MOD: str = 'before'
-    # "after" -> To mean "following in time". I.e. Does not include the extracted datetime entity in the resolution's starting point. Equivalent to ">"
+    # "after" -> To mean "following in time". I.e.
+    # Does not include the extracted datetime entity in the resolution's starting point. Equivalent to ">"
     AFTER_MOD: str = 'after'
     # "since" -> Same as "after", but including the extracted datetime entity. Equivalent to ">="
     SINCE_MOD: str = 'since'
     # "until" -> Same as "before", but including the extracted datetime entity. Equivalent to "<="
     UNTIL_MOD: str = 'until'
     EARLY_MOD: str = 'start'
     MID_MOD: str = 'mid'
+    MID_NIGHT_MOD: str = 'midnight'
```

Comment:
Are these two really constants? Shouldn't them be in the resource files?
I'll try to also double-check this later.

Reply:
They should, for the sake of this implementation, we just added them to the constants file to follow Pyhon's code standards and avoid strings hardcoded through the code. 
We could, as part of a new PR, create the necessary Patterns to have this included at the resources file. We will be opening an issue to modify resources to include Holydays and avoid hardcoded strings.

Reply:
OK, but we should be careful about naming. MOD, GROUP, etc each have different meanings.
For example, is there any case where the output actually includes an entity with MOD attribute with value "midnight"? If not, than the constant should not be named MOD.
If some constants are temporary, a comment in the constants.py file for those is enough for now.

Reply:
We'll add the comment and recheck the constants naming to make sure they are named correctly for when we add them to the resource file.